docs/modules/ROOT/pages/community.adoc

```diff
@@ -21,6 +21,8 @@ We welcome your involvement in the Spring Security project.
 There are many ways to contribute, including answering questions on Stack Overflow, writing new code, improving existing code, assisting with documentation, developing samples or tutorials, reporting bugs, or simply making suggestions.
 For more information, see our https://github.com/spring-projects/spring-security/blob/main/CONTRIBUTING.adoc[Contributing] documentation.
 
+NOTE: Avoid usage of the Stream API for performance reasons. See https://github.com/spring-projects/spring-security/issues/7154[Spring Issue #7154] for more details.
```

Comment:
This was the old guidance. Let's update the guidance to:

```
Since the Stream API is https://github.com/spring-projects/spring-security/issues/7154[much slower] than for loops, please use them judiciously.
The team may ask you to change to a for loop if the given code is along a hot path.
```